stout/os.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <thread>
+
+#include "glog/logging.h" // NOTE: must be included before <windows.h>.
+
+#ifdef __MACH__
+#include <limits>
+#elif _WIN32
+#include <windows.h>
+#else
+#include <pthread.h>
+#endif // __MACH__
+
+////////////////////////////////////////////////////////////////////////
+
+namespace stout {
+
+////////////////////////////////////////////////////////////////////////
+
+#ifdef __MACH__
+inline size_t GetRunningCore() {
```

Comment:
Minor nit but we use both the term "core" and the term "cpu" here, let's just stick to one so that folks understand in our API that we're referring to the same thing! For example:

```c++
inline size_t GetRunningCpu();
```